homeassistant/components/openweathermap/weather_update_coordinator.py

```diff
@@ -182,7 +182,9 @@ def _convert_forecast(self, entry):
     @staticmethod
     def _fmt_dewpoint(dewpoint):
         if dewpoint is not None:
-            return round(dewpoint / 100, 1)
+          """ default metric is kelvin - lets convert"""
+          dewpoint = float(dewpoint - 273.15)
```

Comment:
We don't need to copy to float. It will already be a float.
```suggestion
          dewpoint = dewpoint - 273.15
```

Reply:
We can use this util to convert from Celsius to Fahrenheit:
https://github.com/home-assistant/core/blob/bc3e5b39ed402824b07a29766bf11df9bb8284a3/homeassistant/util/temperature.py#L17-L18

This util is also exposed in our unit system on `hass.config.units`:
```py
dewpoint = self.hass.config.units.temperature(dewpoint, TEMP_CELSIUS)
```

And we can check for the current unit system like this:
```py
if not self.hass.config.units.is_metric:
```

Reply:
Took a look into the code of .temperature() - it will already check for the target metric and convert it currectly, so we can skip the if, isnt it?